lib/mini_tarball/writer.rb

```diff
@@ -38,30 +38,58 @@ def initialize(io)
       @placeholders = []
     end
 
-    def add_file(name:, source_file_path:, mode: nil, uname: nil, gname: nil, uid: nil, gid: nil, mtime: nil)
-      ensure_not_closed!
+    # :reek:ControlParameter
+    # :reek:DuplicateMethodCall { allow_calls: ['stat.uid', 'stat.gid'] }
+    # :reek:FeatureEnvy
+    # :reek:LongParameterList
+    # :reek:TooManyStatements
```

Comment:
Seeing all these control comments: is `reek` providing any value? 😅

Reply:
@CvX Yeah, it was actually quite useful for noticing code smells. But it really hates how I implemented stuff in `MiniTarball::Writer`. :smile: But that's okay...